repose-aggregator/components/filters/saml-policy-translation-filter/src/main/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilter.scala

```diff
@@ -50,16 +61,21 @@ import scala.language.postfixOps
 @Named
 class SamlPolicyTranslationFilter @Inject()(configurationService: ConfigurationService,
                                             atomFeedService: AtomFeedService,
-                                            akkaServiceClientFactory: AkkaServiceClientFactory)
+                                            akkaServiceClientFactory: AkkaServiceClientFactory,
+                                            @Value(ReposeSpringProperties.CORE.CONFIG_ROOT) configRoot: String)
   extends AbstractConfiguredFilter[SamlPolicyConfig](configurationService)
-  with LazyLogging
-  with AtomFeedListener {
+    with LazyLogging
+    with AtomFeedListener {
 
   override val DEFAULT_CONFIG: String = "saml-policy.cfg.xml"
   override val SCHEMA_LOCATION: String = "/META-INF/config/schema/saml-policy.xsd"
 
   private var cache: LoadingCache[String, XsltExecutable] = _
   private var feedId: Option[String] = None
+  private var fac: XMLSignatureFactory = _
```

Comment:
I should have changed these from the original examples during the spike. Updated now.